MODULE.bazel

```diff
@@ -168,6 +168,10 @@ npm.npm_translate_lock(
             "requires-network",
         ],
     },
+    lifecycle_hooks_use_default_shell_env = {
```

Comment:
are these just a couple arbitrary packages to make the feature load-bearing and kinda-tested in our repo? Or is there something about `fsevents` and `rollup`? A comment here would be nice.

Reply:
Just to test the `js_run_binary` generation really. I can look into adding a real test which depends on the latest bazel-lib etc.

Reply:
I'm fine with the answer is "a couple arbitrary ones to trigger the generation" - don't need to take the time of reproducing a bug here

Reply:
I'll move the tests into one of the e2e tests with the latest bazel-lib, I don't want to update the main bazel-lib dep atm.